src/test/java/org/opentripplanner/raptor/moduletests/A01_SingeRouteTest.java

```diff
@@ -68,37 +66,22 @@ public void setup() {
     ModuleTestDebugLogging.setupDebugLogging(data, requestBuilder);
   }
 
-  @Test
-  public void standardOneIteration() {
-    var request = requestBuilder
-      .profile(RaptorProfile.STANDARD)
-      .searchParams()
-      .searchOneIterationOnly()
+  static List<RaptorModuleTestCase> testCases() {
```

Comment:
Not that i makes a huge difference but you can save two lines by removing the method and using a `VariableSource`.

Reply:
I like the method better than the variable - it becomes a bit theoretical. You can improve the reading by replacing `() { ... }` with ` = ...`, but the declarations is no longer in the natural oder (unless you do *not* follow the coding conventions). This is also safer, because you eliminate the problem with oder for static initialization - which I run into in an early stage of this. I think I will leave it, so the definition of the test-case is right before each test - and not on top of the class.